src/pages/card.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./card.css";
+import { SlLogin } from "react-icons/sl";
+
+function Card() {
+  const content = 150;
+  const label = " Since Last Month";
+
+  const getArrowDirection = () => {
+    return content >= 150 ? "up" : "down";
+  };
+
+  const getArrowColor = () => {
+    return getArrowDirection() === "up" ? "#009E0F" : "#CC0000";
+  };
+
+  return (
+    <div className="cardWrap">
```

Comment:
We are already using a component library called `react-bootstrap`, we can use some of their existing component to make our own. For example

https://react-bootstrap.netlify.app/docs/components/cards